model/Core/Vocabularies/RelationshipType.md

```diff
@@ -72,13 +72,15 @@ name completes the sentence:
 - hasTestCase: Every `to` Element is a test case for the `from` Element (`from` hasTestCase `to`).
 - hasVariant: Every `to` Element is a variant the `from` Element (`from` hasVariant `to`).
 - invokedBy: The `from` Element was invoked by the `to` Agent, during a LifecycleScopeType period (for example, a Build element that describes a build step).
+- instantiation: The `to` element is instantiated by the `From` Process element 
```

Comment:
```suggestion
- instantiatedBy: The `to` Element is instantiated by the `from` Process.
```

Other relationships are verbs:

- `to` modifiedBy `from`
- `to` packagedBy `from`
- `to` instantiatedBy `from`